src/app/content/components/Topbar/TextResizer.tsx

```diff
@@ -57,7 +57,7 @@ export const TextResizer = (props: TextResizerProps) => {
       dataAnalyticsLabel='Change text size'
       {...props}
     >
-      <Styled.TextResizerMenu tabIndex={0} bookTheme={props.bookTheme} textSize={props.textSize}>
+      <Styled.TextResizerMenu tabIndex={-1} bookTheme={props.bookTheme} textSize={props.textSize}>
```

Comment:
Does it need a `tabIndex` setting at all? It's a div, which normally will not receive focus. Does this div receive focus programmatically?

Reply:
You're right. For some reason I thought 0 was the default, but that would mean that all non-interactive elements would be in the tab list, so I am not sure what I was thinking.